charts/fluent-operator/Chart.yaml

```diff
@@ -15,7 +15,7 @@ description: A Helm chart for Kubernetes
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.1
+version: 2.1.2
```

Comment:
version can be the same as 2.2.0

Reply:
Should I wait for version 2.2.0 to be released and then update the field to 2.2.0?

Reply:
we're going to release v2.2.0 after this PR, so it's ok to change it now

Reply:
fixed